api_client/python/timesketch_api_client/search.py

```diff
@@ -249,6 +249,11 @@ def add_end_time(self, end_time):
         """
         if end_time.endswith("Z"):
             end_time = end_time[:-1]
+
+        # Check for a whole day, YYYY-MM-DD.
+        if len(end_time) == 10:
```

Comment:
Do we want to check for the format here as well, and not just the length of the string?
`YYYY-MM-DD`

Reply:
using re